### PR TITLE
Feat/remove class name prop

### PR DESCRIPTION
**Pull request by andresin87**
remove some className props not needed

### CODE REVIEW

File: components/atom/tag/src/index.js [hunk 1]
```diff
@@ -80,7 +80,6 @@ AtomTag.propTypes = {
   /**
    * Custom classes
```
Comment (jcger) [inline components/atom/tag/src/index.js:81]: I'd say this comment should be removed too, looks like its referring the className prop

**Comment by jcger** [comment]
I can't see no breaking changes defined in your commit messages so this would be released as a minor, you sure about this?

**Comment by andresin87** [comment]
> I can't see no breaking changes defined in your commit messages so this would be released as a minor, you sure about this?

I think so. The prop is not used in the component for both cases. I would reword the commits to do not force a major
